meson_options.txt

```diff
@@ -12,6 +12,9 @@ option('enable_asserts', type: 'boolean', value: false,
 option('enable_docs', type: 'boolean', value: false,
     description: 'build documentation')
 
+option('max_tailqs', type: 'integer', value: '32',
```

Comment:
Does this need to be configurable? I think 32 is probably enough, if we need it to be bigger, we can just change the \#define.